microprofile/health-smallrye/src/main/java/org/wildfly/extension/microprofile/health/_private/MicroProfileHealthLogger.java

```diff
@@ -52,4 +52,12 @@ public interface MicroProfileHealthLogger extends BasicLogger {
     @Message(id = 6, value = "")
     OperationFailedException seeDownstream();
     */
+
+    @LogMessage(level = WARN)
+    @Message(id = 7, value = "Disabling default procedures according to the %s deployment configuration")
+    void disablingDefaultProcedures(String deploymentName);
```

Comment:
If this is spec defined behavior that we document as being acceptable for a microservice-style installation, it seems wrong to WARN about it.

Reply:
+1, thanks. Now there's just one WARN that we output upon the problematic condition you described in the review summary.

Reply:
And now changed in to an `INFO`, resolving.